API/Controllers/ProjectController.cs

```diff
@@ -239,6 +239,26 @@ public async Task<IActionResult> MigrateDatabaseToElasticSearch()
             return Ok(resultsResource);
         }
 
+        /// <summary>
+        ///     This method returns suggestions while searching for projects
+        /// </summary>
+        /// <param name="query"></param>
+        /// <returns>This method returns a list of autocomplete project resources.</returns>
+        /// <response code="200">This endpoint returns a list with suggested projects.</response>
+        [HttpGet("search/autocomplete")]
+        [ProducesResponseType(typeof(AutocompleteProjectsResource), (int) HttpStatusCode.OK)]
+        [ProducesResponseType(typeof(ProblemDetails), (int) HttpStatusCode.BadRequest)]
```

Comment:
In what scenario does this method return a ProblemDetails object with status code BadRequest? I think this can be removed.